src/content/en/2020/cms.md

```diff
@@ -23,3 +22,364 @@ published: 2020-12-09T00:00:00.000Z
+  sheets_gid="465267881",
+  sql_file="core_web_vitals_distribution.sql"
+  )
+}}
+
+High page weights are a possible explanation here; we’ll explore this, and the possible impact of this on the environment in a later section. It is clear here, however, that LCP performance is generally bad for CMSs, and the impact on the user experience is also bad.
+
+**First Input Delay**
+
+First Input Delay (FID) measures the time from when a user first interacts with your site (i.e. when they click a link, tap on a button, or use a custom, JavaScript-powered control) to the time when the browser is actually able to respond to that interaction. A "fast" FID from a user's perspective would be immediate feedback from their actions on a site rather than a stalled experience. This delay (a pain point) could correlate with interference from other aspects of the site loading when the user tries to interact with the site.
+
+FID is very fast for the average CMS website on desktop – only Wix scores lower than 100% – and mixed on mobile. Most CMSs deliver mobile FID on an average site within a reasonable range of the desktop score; for Wix the number of websites that have a good FID on mobile is nearly half the desktop total.
+
+{{ figure_markup(
+  image="cms-real-user-first-input-delay-experiences.png",
+  caption="Real-user First Input Delay experiences.",
+  description='Bar chart showing the top 5 CMSs and whether they have a "good" First Input Delay experience. All have a 100% experience score on desktop, except Wix with 87%. For mobile Wordpress has 88%, Drupal 76%, Joomla 71%, Squarespace 91% and Wix 46%.',
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vTkUxfuK-FCn3_IYDJiEsKDmdmyKb0TSEhG8dFc0XBIXej1NO2uUOmd-9NCbWuh-MZ3xzMhK_kNT-4u/pubchart?oid=893606466&format=interactive",
+  sheets_gid="465267881",
+  sql_file="core_web_vitals_distribution.sql"
+  )
+}}
+
+The FID scores are generally good here, but this does not translate to similar LCP scores. As suggested, the weight of individual pages on CMSs in addition to mobile connection quality, could play a role in the performance gaps that we see here. There is a small margin of difference between the resources shipped to desktop and mobile versions of a website. Last year we noted that optimizing for the mobile experience was necessary. Average scores have increased on desktop and mobile, but further attention is required here.
+
+**Cumulative Layout Shift**
+
+Cumulative Layout Shift (CLS) measures the instability of content on a web page within the first 500ms of user input. CLS measures any layout changes which happen after user input. This is important on mobile in particular, where the user will tap where they want to take an action – such as a search bar – only for the location to move as additional images, ads, or similar load.
```

Comment:
CLS actually [_excludes_](https://web.dev/cls/#user-initiated-layout-shifts:~:text=500%20milliseconds) any layout shifts occurring within 500ms of a user interaction. The reason being that UIs are expected to change and react to inputs, but a page's layout shouldn't jump around on its own. Worth rewriting this paragraph.

Reply:
ok, revised. I hadn't heard of CLS before starting this project and got the info for what I wrote from that page! Feel free to make any further changes to this directly if required.